e2e/imgbuild/imgbuild.go

```diff
@@ -14,7 +14,6 @@ import (
 	"testing"
 
 	"github.com/sylabs/singularity/e2e/internal/e2e"
-	"github.com/sylabs/singularity/internal/pkg/test"
 )
 
 var testFileContent = "Test file content\n"
```

Comment:
what's the use of this variable?

Reply:
Looks like a canary content for definition files